db/seeds.rb

```diff
@@ -1,7 +1,15 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+User.create!(first_name: 'Admin',
+             last_name: 'Admin',
+             email: 'admin@example.com',
+             phone: '0',
+             gender: 'None',
+             date_of_birth: '1/1/1970',
+             admin: true)
+
+User.create(first_name: 'Yaroslav',
+            last_name: 'Liakh',
+            email: 'liakhyaroslav@gmail.com',
+            phone: '+380506456220',
+            gender: 'Man',
```

Comment:
Please, correct me if I am wrong, but it will be better to use boolean value for gender